dashboard/src/components/partials/ChecksList/ChecksListItem.js

```diff
@@ -32,7 +32,6 @@ class CheckListItem extends React.Component {
       fragment ChecksListItem_check on CheckConfig {
         name
         command
-        source
```

Comment:
Just confirming this single line removal was intentional?

Reply:
I was. `source` was deprecated causing this line upset the linter.